bankdroid-legacy/src/main/res/raw/cert_paypal.pem

```diff
@@ -1,23 +1,4 @@
 -----BEGIN CERTIFICATE-----
-<<<<<<< HEAD
```

Comment:
Well, it were introduced by you (de9aab907f), I'm just cleaning up your mess :smile: